content/automate/ManageIQ/System/Event/MiqEvent/POLICY.class/__methods__/check_policy_prevent.rb

```diff
@@ -0,0 +1,8 @@
+#
+# Description: This method stops the retirement if the policy prevents
+#
+event = $evm.root['event_stream']
+if event.full_data && event.full_data[:policy][:prevented]
```

Comment:
@mkanoor I think it can reach this point in automate only if the event stream object exists.